src/main/java/org/mule/extensions/jms/api/connection/factory/activemq/ActiveMQConnectionFactoryConfiguration.java

```diff
@@ -170,6 +171,13 @@ public void setTrustAllPackages(boolean trustAllPackages) {
     this.trustAllPackages = trustAllPackages;
   }
 
+  public boolean getVerifyHostName() {
+    //TODO: change the default value to a parameter. This default is required to support a security fix and avoid
+    // breaking backwards compatibility (GUS ticket: W-14487260)
+    return false;
+  }
```

Comment:
As part of the Java 17 initiative we are going to expose this value as a configurable parameter in a new feature and release a new minor version, that's why we are generating a hardcoded setter in the configuration, it will be transformed into a configurable parameter.